Remotion/Web/Development.WebTesting/WebDriver/Configuration/Chromium/IChromiumConfiguration.cs

```diff
@@ -17,29 +17,32 @@
 using System;
 using JetBrains.Annotations;
 using OpenQA.Selenium.Chrome;
+using Remotion.Web.Development.WebTesting.WebDriver.Configuration.Chromium.Chrome;
+using Remotion.Web.Development.WebTesting.WebDriver.Configuration.Chromium.Edge;

```

Comment:
These using directives are only there so we can reference the concrete implementations for Chrome and Edge in the interface documentation. Is that even desirable or wouldn't it be cleaner to remove the interface's knowledge of its concrete implementations altogether?

Reply:
How is that "knowledge"?

But good point, the `seealso`'s are unnecessary.